esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -0,0 +1,242 @@
+"""Fixes for ERA5."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+
+class FixEra5(Fix):
+    """Fixes for ERA5 variables."""
+
+    @staticmethod
+    def _frequency(cube):
+
+        if not cube.coords(axis='T'):
+            return 'fx'
+        time = cube.coord(axis='T')
+        if len(time.points) == 1:
+            return 'fx'
+
+        interval = time.points[1] - time.points[0]
+        unit = 'hours' if 'hour' in time.units.name else 'days'
+        if (unit == 'hours' and interval == 1):
+            return 'hourly'
+        if (unit == 'days' and interval - 1 / 24 < 1e-4):
+            return 'hourly'
+        return 'monthly'
+
+
+class Accumulated(FixEra5):
+    """Fixes for accumulated variables."""
+
+    def _fix_frequency(self, cube):
+        if cube.var_name in ['mx2t', 'mn2t']:
+            pass
+        elif self._frequency(cube) == 'monthly':
+            cube.units = cube.units * 'd-1'
+        elif self._frequency(cube) == 'hourly':
+            cube.units = cube.units * 'h-1'
+        return cube
+
+    def _fix_hourly_time_coordinate(self, cube):
+        if self._frequency(cube) == 'hourly':
+            time = cube.coord(axis='T')
+            time.points = time.points - 0.5
+        return cube
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        super().fix_metadata(cubes)
+        for cube in cubes:
+            self._fix_hourly_time_coordinate(cube)
+            self._fix_frequency(cube)
+        return cubes
+
+
+class Hydrological(FixEra5):
+    """Fixes for hydrological variables."""
+
+    @staticmethod
+    def _fix_units(cube):
+        cube.units = 'kg m-2 s-1'
+        cube.data = cube.core_data() * 1000.
+        return cube
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        super().fix_metadata(cubes)
+        for cube in cubes:
+            self._fix_units(cube)
+        return cubes
+
+
+class Radiation(FixEra5):
+    """Fixes for accumulated radiation variables."""
+
+    @staticmethod
+    def _fix_direction(cube):
+        cube.attributes['positive'] = 'down'
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        super().fix_metadata(cubes)
+        for cube in cubes:
+            self._fix_direction(cube)
+        return cubes
+
+
+class Fx(FixEra5):
+    """Fixes for time invariant variables."""
+
+    @staticmethod
+    def _remove_time_coordinate(cube):
+        cube = iris.util.squeeze(cube)
```

Comment:
```suggestion
        cube = cube[0]
```
may be a bit more readable